lib/bump/bump.ts

```diff
@@ -272,18 +301,26 @@ function createPullRequestCommands(repo: WritableGitHubRepo, options: PullReques
     '{ echo "No changes after bump. Skipping pull request..."; export SKIP=true; } || ' +
     '{ echo "Creating pull request..."; export SKIP=false; }';
 
-  const curl = [
-    `curl`,
-    `-X POST`,
-    `--header "Authorization: token $GITHUB_TOKEN"`,
-    `--header "Content-Type: application/json"`,
-    `-d ${JSON.stringify(JSON.stringify(request))}`, // to escape quotes
-    `https://api.github.com/repos/${repo.owner}/${repo.repo}/pulls`
+  const commands = [
+     condition,
+    `GITHUB_TOKEN=$(aws secretsmanager get-secret-value --secret-id "${repo.tokenSecretArn}" --output=text --query=SecretString)`,
+    `${curl('/pulls', '-X POST -o pr.json', request, repo)} && export PR_NUMBER=$(node -p 'require("./pr.json").number')`
   ];
```

Comment:
Why write the PR numbers to a file and also export it? The commend that requires the issue number can read it from the file (same way at the export is created)

Reply:
We might have additional commands that need access to the PR number. This way users can also just use `$PR_NUMBER` in their own custom PR body.